File: src/main/java/de/ingrid/igeserver/services/ExportPostProcessors.java
package de.ingrid.igeserver.services;

import com.fasterxml.jackson.databind.JsonNode;

public interface ExportPostProcessors {
	
	public enum TransformationType {
		ISO
	}

	public Object process(Object exportedDoc, JsonNode jsonData);
	
	public TransformationType getType();
	
}

File: server/src/main/java/de/ingrid/igeserver/services/ExportPostProcessors.kt
package de.ingrid.igeserver.services;

import com.fasterxml.jackson.databind.JsonNode;

public interface ExportPostProcessors {
	
	enum TransformationType {
		ISO
	}

	Object process(Object exportedDoc, JsonNode jsonData);
	
	TransformationType getType();
	
}
